spk/synocli-file/Makefile

```diff
@@ -1,37 +1,30 @@
 SPK_NAME = synocli-file
 SPK_VERS = 1.0
```

Comment:
I propose to increase package version itself too
```suggestion
SPK_VERS = 1.1
```

Reply:
My intent is to update revision for updated components and update the version with adding/removing components.
But we could use major/minor package version for this.

Reply:
I prefer the other way "major/minor" to get a reason to increase major from time to time...